utils/check_repo.py

```diff
@@ -210,7 +210,6 @@
     "ChineseCLIPVisionModel",
     "CLIPTextModel",
     "CLIPTextModelWithProjection",
-    "CLIPVisionModel",
```

Comment:
FMI could you explain this one?

Reply:
CLIPVisionModel is now supported by `AutoModel`: https://github.com/huggingface/transformers/blob/main/src/transformers/models/auto/modeling_auto.py#L62

Reply:
Good catch 👍🏻